### PR TITLE
Adds RollbarBackend and fixes MultiBackend

### PR DESCRIPTION
**Pull request by v-yarotsky**
@sumeet I mostly ported over your code here

### CODE REVIEW

File: lib/exceptions/backends/rollbar.rb [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+      def notify(exception = nil, level: 'error', error_class: nil, error_message: nil, parameters: {}, context: {})
+        err = exception || error_class
+        extra = [DEFAULT_NOTIFY_ARGS, parameters, context].reduce(&:merge)
+        Response.wrap(rollbar.log(level, err, error_message, extra))
+      end
+
+      def context(ctx)
+        # rollbar does special tracking of the `person` key, but `user` is a more
+        # common name for this.
+        ctx = Marshal.load(Marshal.dump(ctx)) # deep clone
```
Comment (ejholmes) [inline lib/exceptions/backends/rollbar.rb]: I wonder if there are any side effects of doing this. Maybe performance?

File: lib/exceptions/backends/rollbar.rb [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+      end
+
+      def notify(exception = nil, level: 'error', error_class: nil, error_message: nil, parameters: {}, context: {})
+        err = exception || error_class
+        extra = [DEFAULT_NOTIFY_ARGS, parameters, context].reduce(&:merge)
+        Response.wrap(rollbar.log(level, err, error_message, extra))
+      end
+
+      def context(ctx)
+        # rollbar does special tracking of the `person` key, but `user` is a more
```
Comment (ejholmes) [inline lib/exceptions/backends/rollbar.rb]: I think you want this comment one line lower.

**Comment by sumeet** [comment]
omg thanks so much for pulling this out, and adding tests! looks great 👍

File: lib/exceptions/backends/rollbar.rb (hunk 1 above)
Comment (sumeet) [inline lib/exceptions/backends/rollbar.rb]: this was to break the dependency on activesupport? :)

Comment (v-yarotsky) [inline lib/exceptions/backends/rollbar.rb]: yea

Comment (v-yarotsky) [inline lib/exceptions/backends/rollbar.rb]: I just removed this deep clone hack since shallow dup is enough for the thing I'm trying to achieve

**Pull request merged**